hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -100,6 +118,7 @@ public boolean match(Event value) {
     private static final String ID = "id";
     private static final String CTIME = "ctime";
     private static final String TEXT = "text";
+    private static final String CATEGORY = "category";
     private static final String CONTEXT = "context.";
```

Comment:
I agree that Events are not standard Data, but Context is available on Data, and on Triggers etc, and is never available for filtering.  I think we will send a strange, inconsistent message if we make it available here, just because we can.